Tests/ResponsesSpec/httpResponseSpec.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import Quick
+import Nimble
+import Responses
+
+class HttpResponseSpec: QuickSpec {
+    override func spec() {
+        describe ("#HttpResponse") {
+            it ("can equate two response objects") {
+                let response1 = HttpResponse(
+                        version: "HTTP/1.1",
+                        statusCode: 200,
+                        statusPhrase: "OK",
+                        headers: [:],
+                        body: ""
+                )
```

Comment:
Would it be possible to create a test helper that creates a default `HttpResponse`? That way, you would not have to write it out over and over again. Same for `HttpRequest` maybe